# 9D加速度センサSET
float64 accX
float64 accY
float64 accZ
---

